hack/go-version-ok

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euo pipefail
+GO=${GO:-go}
+
+if [[ $(($($GO version | sed -n 's/.*go1\.\([0-9]\+\).*/\1/p'))) -gt 10 ]]; then
```

Comment:
could we test for the feature instead of testing for the version?

Would something like `go help build | grep -- -mod` work in this case?

Reply:
The current check does not look forward compatible with Go 2.

Something like `go version | awk '{ printf "%s\ngo1.11.0", $3 }' | sort -V` and then checking if `go1.11.0` is first could be more stable.

Reply:
Or simpler with `go help mod >> /dev/null` and checking the exit code.

Reply:
> Or simpler with `go help mod >> /dev/null` and checking the exit code.

Okay, I'd chose this variant and removed the script since it is now that simple. Thanks!